guide/plugins_apollocar3d.ipynb

```diff
@@ -0,0 +1,158 @@
+{
+ "cells": [
+  {
+   "source": [
+    "# Car Keypoints\n",
+    "\n",
+    " <div style=\"text-align: right\"> by <a href=\"https://scholar.google.com/citations?user=f-4YHeMAAAAJ&hl=en\">Lorenzo Bertoni </a> and Duncan Zauss, 13/04/2021 </div> <br /> \n",
+    "\n",
+    "\n",
+    "\n",
+    "This section describes the [OpenPifPaf](https://github.com/openpifpaf/openpifpaf) plugin for vehicles. The plugin uses the [ApolloCar3D Dataset](http://apolloscape.auto/car_instance.html). For more information, we suggest to check our latest [paper](https://arxiv.org/abs/2103.02440): <br /> \n",
+    "\n",
+    "> __OpenPifPaf: Composite Fields for Semantic Keypoint Detection and Spatio-Temporal Association__<br />\n",
+    "> _[Sven Kreiss](https://www.svenkreiss.com), [Lorenzo Bertoni](https://scholar.google.com/citations?user=f-4YHeMAAAAJ&hl=en), [Alexandre Alahi](https://scholar.google.com/citations?user=UIhXQ64AAAAJ&hl=en)_, 2021.\n",
+    ">\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "```sh\n",
+    "pip3 install openpifpaf\n",
+    "```\n",
+    "\n",
+    "(in case CUDA 9 as driver: \n",
+    "` pip install torch==1.7.0+cu92 torchvision==0.8.1+cu92 -f https://download.pytorch.org/whl/torch_stable.html`)\n",
+    "\n",
+    "## Predict \n",
+    "Prediction runs as standard openpifpaf predict command, but using the pretrained model on vehicles. The flag \"--checkpoint shufflenetv2k16-apollo-24\" will cause that our 24 kp version of the Shufflenet 16 (AP 76.1%) will be automatically downloaded. As an example, run the command:\n",
+    "\n",
+    "```sh\n",
+    "python -m openpifpaf.predict \\\n",
+    "<image path> \\\n",
+    "--checkpoint shufflenetv2k16-apollo-24 -o \\\n",
+    "--instance-threshold 0.07 --seed-threshold 0.07 \\\n",
+    "--line-width 3 --font-size 0 --white-overlay 0.6 \n",
+    "```\n",
+    "\n",
+    "## Preprocess Dataset\n",
+    "### Keypoints choice\n",
+    "The preprocessing step converts the annotations into the standard COCO format. It creates a version with all 66 keypoints and also creates a sparsified version with 24 keypoints. The resulting pose can be obtained running:\n",
+    "\n",
+    "```sh\n",
+    "python -m openpifpaf_apollocar3d.utils.constants\n",
+    "```\n",
+    "\n",
+    "GIFS TO ADD\n",
```

Comment:
@svenkreiss,  @DuncanZauss can I get your inputs on creating the gifs on the fly?

Reply:
Hi @bertoni9,
To create the gifs with the current pifpaf version I run (from the top-level openpifpaf folder):
mkdir openpifpaf/plugins/apollocar3d/docs/
And then:
python -m openpifpaf.plugins.apollocar3d.apollocar3d.constants
After that the gifs will be in openpifpaf/plugins/apollocar3d/docs/.
Is that helpful?